pkg/daemon/daemon.go

```diff
@@ -1283,7 +1284,7 @@ func (dn *Daemon) prepUpdateFromCluster() (*mcfgv1.MachineConfig, *mcfgv1.Machin
 // "transient state" file, which signifies that all of those prior steps have
 // been completed.
 func (dn *Daemon) completeUpdate(node *corev1.Node, desiredConfigName string) error {
-	if err := drain.RunCordonOrUncordon(dn.drainer, node, false); err != nil {
+	if err := dn.cordonOrUncordonNode(false); err != nil {
```

Comment:
Since you use the `dn.Node` reference instead, the linter is complaining the `node` variable isn't used anymore here

Reply:
yeah, fixed it